spk/ffsync/src/requirements.txt

```diff
@@ -92,11 +92,11 @@ repoze.lru==0.6
 #requests==2.4.3
 simplejson==3.6.5
 six==1.8
-testfixtures==4.1.0
```

Comment:
This was failing building. But since it's related to testing I figured we could dismiss it, right ?